tests/testthat/test-compare-character.R

```diff
@@ -71,3 +71,11 @@ test_that("not all lines are shown", {
   expect_equal(lines[1], "1/1 mismatches")
   expect_equal(length(lines), 8)
 })
+
+test_that("longer vectors (#513)", {
+  expect_warning(comp <- compare(letters[1:6], LETTERS[1:6]), NA)
```

Comment:
The code this PR fixes used to produce a warning when there are more than 5 mismatches.  I guess this is not strictly needed going forward assuming no one re-introduces the same error, but thought it might make sense to try to avoid an accidental regression.

Please confirm that you want me to remove it.